noxfile.py

```diff
@@ -11,6 +11,9 @@ def tests(session: nox.Session) -> None:
 
 @nox.session(python="3.11")
```

Comment:
Similarly tests should capture Python 3.8-3.11 at least, I believe it is sufficient to call nox only in 3.11 though.